macOS/LocalPackages/NetworkProtectionMac/Sources/VPNAppState/VPNAppState.swift

```diff
@@ -87,4 +88,20 @@ public final class VPNAppState {
     public func resetIsAuthV2Enabled() {
         defaults.resetIsAuthV2Enabled()
     }
+
+    // MARK: - VPN Exclusion Suggestion when turning it OFF
+
+    public var dontAskAgainExclusionSuggestion: Bool {
```

Comment:
`VPNAppState` is a reasonable place to put this - it serves the purpose of storing app state across the main app and the VPN menu app so they can stay synchronized.  While we're not using this in the menu app, `VPNAppState` is will the default place to store this kind of configuration.